src/index.js

```diff
@@ -1,26 +1,25 @@
-import { leaderboard, UpdateLocalStorage } from './modules/data.js';
+// import { leaderboard } from './modules/data.js';
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.